### DISCUSSION TITLE
Coach Funktionen

### DISCUSSION THREAD

**Discussion by Mundus-Steffen** [berliCRM/berlicrm#886] [General]
Hallo! Findet sich hier vielleicht jemand, der bereit ist, mich zu Basis-Funktionen des Berli-CRM zu informieren? Bin bei einem Arbeitgeber gelandet, der Berli nutzt, aber es kennt sich niemand wirklich damit aus... insofern wird das System auch noch nicht so sinnvoll genutzt. Das würde ich gerne ändern.... Gibts vielleicht jemanden, der Lust hätte, dazu mal zu schreiben oder zu telefonieren...? Ich würde mich SEHR freuen! LG Steffen

**Comment by Archibald111** [comment]
ja sicher, einfach eine Mail an servicedesk@crm-now.de schicken und nach einen Termin fragen

**Comment by Mundus-Steffen** [comment]
Hi Frank,

vielen Dank, mache ich so!

Von: Frank Piepiorra ***@***.***>
Gesendet: Freitag, 29. November 2024 09:51
An: berliCRM/berlicrm ***@***.***>
Cc: Steffen Schott ***@***.***>; Author ***@***.***>
Betreff: Re: [berliCRM/berlicrm] Coach Funktionen (Discussion #886)


ja sicher, einfach eine Mail an ***@***.******@***.***> schicken und nach einen Termin fragen
